src/components/Calculator.js

```diff
@@ -1,55 +1,163 @@
 /** @format */
 
-import React from 'react';
-import { Table } from 'reactstrap';
-import './calculator.css';
+import React, { Component } from 'react';
+import { Table, Button } from 'reactstrap';
+import Calculate from '../logic/calculate';
+import './Calculator.css';
+
+export default class Calculator extends Component {
```

Comment:
- [ OPTIONAL ] In order to learn more about props I would recommend creating a new `button` component where you will do the calculation and pass it the state there as a prop from `Calculator`